extensions/rich_text_components/Image/Image.py

```diff
@@ -47,9 +48,19 @@ class Image(base.BaseRichTextComponent):
         'default_value': '',
     }, {
         'name': 'alt',
-        'description': 'Alternative text (for screen readers)',
+        'description': (
+            'Briefly explain this image to a visually impaired '
+            'learner'),
         'schema': {
             'type': 'unicode',
+            'validators': [{
+                'id': 'is_nonempty',
+            }],
+            'ui_config': {
+                'placeholder': (
+                    'Description of Image (Example : "George Frideric, '
```

Comment:
@anthkris I removed the Handel from George Frideric Handel because that was overflowing the input field

![screenshot from 2017-02-01 17 53 35](https://cloud.githubusercontent.com/assets/13452436/22506642/9883f12e-e8a7-11e6-8fa6-c52bbbc50232.png)

Can we remove Description of image and just keep  this part
Example : George Frideric Handel, 18th century baroque composer